config/rector.php

```diff
@@ -11,9 +16,39 @@
             __DIR__ . '/../tests',
         ]
     )
-    ->withPhpSets()
-    ->withRules(
-        [
-            // AddVoidReturnTypeWhereNoReturnRector::class,
-        ]
-    );
+    ->withPhpVersion(PhpVersion::PHP_73)
+    ->withPhpSets(
+        true
+    )
```

Comment:
I can't find any documentation for this having a simple parameter of `true`.  All I found was [this](https://getrector.com/documentation/set-lists).  So it doesn't seem right.

In any case, the parameter should be on the same line as the method call.

Reply:
Removed.